ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -950,7 +950,8 @@ public void onDataNodeConfigChange(Iterable<DataNodeConfig> configs) {
           } finally {
             instanceConfigInitialized = true;
           }
-          sealedStateChangeCounter.incrementAndGet();
+          long counter = sealedStateChangeCounter.incrementAndGet();
+          logger.trace("SealedStateChangeCounter increase to {}", counter);
```

Comment:
nit: this could be simplified to 
`logger.trace("SealedStateChangeCounter increase to {}", sealedStateChangeCounter.incrementAndGet());`